pkg/identityManager/const.go

```diff
@@ -28,6 +26,8 @@ const (
 	remoteClusterIDTagKey = "liqo.io/remote-cluster-id"
 	managedByTagKey       = "liqo.io/managed-by"
 	managedByTagValue     = "liqo"
+	identityTypeTagKey    = "liqo.io/identity-type"
```

Comment:
We could re-use the identical label in `pkg/consts/authentication.go`

Reply:
I copied the value from this constant to the other, in order to continue using the tagkey suffix, is it ok for you?